qiskit/result/distributions/quasi.py

```diff
@@ -123,3 +123,11 @@ def hex_probabilities(self):
                 format ``"0x1a"``
         """
         return {hex(key): value for key, value in self.items()}
+
+    def stddev(self, shots):
```

Comment:
This does not work here.  The runtime mitigation method will soon return the `mitigation_overhead` that, together with the number of shots, gives an estimate (usually exact) of this value.

Reply:
Thank you @nonhermitian for the feedback! So where should I add the `stddev` function? 
Should I add it to the ReadoutMitigator class?

Reply:
It makes sense but needs to be computed differently.